packages/expo-file-system/ios/EXFileSystem/EXFileSystem.m

```diff
@@ -702,16 +761,30 @@ - (NSMutableDictionary *)_parseServerResponse:(NSURLResponse *)response
   return result;
 }
 
-- (NSString *)_importHttpMethod:(NSNumber *)httpMethod
+- (NSURLSession *)_createSessionWithHeaders:(NSDictionary * _Nullable)headers
+{
+  NSURLSessionConfiguration *sessionConfiguration = [NSURLSessionConfiguration backgroundSessionConfigurationWithIdentifier:[[NSUUID UUID] UUIDString]];
+  sessionConfiguration.requestCachePolicy = NSURLRequestReloadIgnoringLocalCacheData;
+  if (headers != nil) {
+    sessionConfiguration.HTTPAdditionalHeaders = headers;
+  }
+  sessionConfiguration.URLCache = nil;
+  
+  return [NSURLSession sessionWithConfiguration:sessionConfiguration];
+}
+
+- (NSString *)_importHttpMethod:(NSNumber * _Nullable)httpMethod
 {
   if ([httpMethod isEqual:@(EXFileSystemPutMethod)]) {
     return @"PUT";
   }
-  if ([httpMethod isEqual:@(EXFileSystemPatchMethod)]) {
-    return @"PATCH";
-  }
-  
-  return @"POST";
+
+  return [httpMethod isEqual:@(EXFileSystemPatchMethod)] ? @"PATCH" : @"POST";
+}
+
+- (BOOL)_isRawUploadType:(NSNumber * _Nullable)uploadType
+{
+  return ![uploadType isEqual:@1];
```

Comment:
I dislike this one .. 😞 
`1)` I saw that you decided on naming it differently than `RAW` (`BINARY_CONTENT` 🤔)
`2)` This magic number `1` is like an eyesore 😭